lib/changeable.rb

```diff
@@ -14,6 +14,10 @@ def changeable?(user)
     elsif user.has_role?("admin", "manager")
       true
 
+    # project admins can change the projects they are admins of
+    elsif (self.class == Admin::Project) && user.is_project_admin? && user.admin_for_projects.include?(self)
+      true
+
```

Comment:
Seems this should be moved into an `Admin::Project.changeable?` that overrides this one and calls super.

In the future I think we should avoid using Changeable as we shift to policies. But that is a bigger change so lets stick with Changeable for now.